### PR TITLE
Add BioPhi first release

### PR DESCRIPTION
**Pull request by prihoda**
BioPhi is an open-source antibody design platform: https://github.com/Merck/BioPhi

BioPhi has a command-line and web interface and can be used to process antibody sequences.

### CODE REVIEW

**Comment by prihoda** [comment]
@BiocondaBot please add label

File: recipes/biophi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - pip
```
Comment (dpryan79) [inline recipes/biophi/meta.yaml:25]: Does it need pip and pytest to run?

Comment (prihoda) [reply recipes/biophi/meta.yaml]: Good catch, I removed pip from here, and I moved pytest to host section

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**